lib/client/api.go

```diff
@@ -1562,51 +1562,25 @@ func (tc *TeleportClient) Join(ctx context.Context, mode types.SessionParticipan
 		return trace.Wrap(err)
 	}
 
-	// find the session ID on the site:
-	sessions, err := site.GetSessions(namespace)
+	var session types.SessionTracker
+	sessions, err := site.GetActiveSessionTrackers(ctx)
```

Comment:
@xacrimon With this change, will a `v9` `tsh` client fail to join a `v8` node session, since node sessions only create session trackers as of `v9` (And later for proxy ssh sessions)?

Reply:
Yup, this requires a special backport for v9. It won't be directly backport in this state.

Reply:
But I am applying it here to master since v9 clients will look for this and thus will be able to join v10 auths.

Reply:
The one backported to v9 will use the ping versioning information about the auth to figure out which tracker type to request.

Reply:
I also plan to phase out the old rsession metadata system entirely at some point.